app/controllers/publishers/base_controller.rb

```diff
@@ -3,6 +3,6 @@ class Publishers::BaseController < ApplicationController
                 :check_terms_and_conditions
 
   def check_terms_and_conditions
-    redirect_to terms_and_conditions_path unless current_publisher.accepted_terms_and_conditions?
+    redirect_to terms_and_conditions_path unless current_publisher.accepted_terms_at?
```

Comment:
Can I be extra cheeky and ask to put this mini 🧹 🔥  in a separate PR?

Reply:
of course you can!